docs/user/aws/images/Makefile

```diff
@@ -0,0 +1,20 @@
+all: install_upi.svg install_upi_vpc.svg
```

Comment:
can we document how to build it / or a small script rather than add make for consistency?

Reply:
> can we document how to build it / or a small script rather than add make for consistency?

We could, but it's going to take more space than the `Makefile` (e.g. we'd have to write our own code or text about checking for a pre-existing icon zip, unpacking the zip, etc.  I don't expect a huge contributor base feeding diagram changes (although I guess you never know), so I'm ok with it how it stands, but am also happy to add docs (`CONTRIBUTING.md`?  `docs/user/aws/images/README.md`?  A `Makefile` header comment?) that say:

> To rebuild the `*.svg`, run `make` in this directory.